molo/core/content_import/validation.py

```diff
@@ -13,10 +13,16 @@ def __init__(self, ws):
 
     def is_validate_for(self, locales):
         self.errors = []
+        main_language = None
         for l in locales:
             if l.get('is_main'):
                 main_language = l.get('locale')
                 break
+        if not main_language:
+            self.errors.append({
+                'type': 'no_main_language',
+            })
+            return self.errors
```

Comment:
I think validation it would make more sense for validation to also use `get_main_language` so it can raise an `ImportError` so that we can handle `ImportError`s the same way for both importing and validation. We shouldn't be showing the user this `no_main_language` error like we are for the other errors, since it indicates a *bug* rather than an error related to importing the content that we want to show to the user.